pages/linux/at.md

```diff
@@ -0,0 +1,15 @@
+# at
+
+> Executes commands at a specified time.
+
+- Open `at` prompt to execute commands at specific time. When you're done, press Ctrl+D to save the schedule:
```

Comment:
I like your proposal @ostera 👍 maybe I'd tweak it to say "create a new set of scheduled commands" or something like that.